### PR TITLE
Feature/web deployment

### PR DESCRIPTION
**Pull request by marcofranssen**
Adds helm deployment for dctna-web component.

### CODE REVIEW

File: templates/server-service.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+kind: Service
+metadata:
+  name: {{ include "dctna.fullname" . }}-server
+  labels:
+    {{- include "dctna.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.server.service.type }}
+  ports:
+    - port: {{ .Values.server.service.port }}
+      targetPort: https
```
Comment (npalm) [inline templates/server-service.yaml:11]: Is the service internally running on HTTPS?

File: templates/web-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ $fullName }}-web
+  labels:
+    {{- include "dctna.labels" . | nindent 4 }}
+  {{- with .Values.web.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.web.ingress.tls }}
+  tls:
```
Comment (npalm) [inline templates/web-ingress.yaml:20]: Not sure if this part already will work on our current set up

File: templates/web-ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - hosts:
+        {{- range .hosts }}
+        - {{ . | quote }}
+        {{- end }}
+      secretName: {{ .secretName }}
+    {{- end }}
+  {{- end }}
+  rules:
+    {{- range .Values.web.ingress.hosts }}
+    - host: {{ .host | quote }}
```
Comment (npalm) [inline templates/web-ingress.yaml:31]: I case you would like to support http -> https you have to add a alias rule, see game example

File: values.yaml [hunk 4]
```diff
@@ -16,3 +14,55 @@ serviceAccount:
...
+    hosts:
+      - host: dctna.philips-labs.local
+        paths: []
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+server:
+  service:
```
Comment (npalm) [inline values.yaml:66]: For AWS we use NodePort

File: templates/server-service.yaml (hunk 1 above)
Comment (marcofranssen) [reply templates/server-service.yaml:11]: yes there is a selfsigned cert.

File: values.yaml (hunk 4 above)
Comment (marcofranssen) [reply values.yaml:66]: Why? The default is ClusterIP.

https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types

In general I don't want a service to be exposed by default. Preferably I put an ingress for things that need to be reachable outside.

**Comment by marcofranssen** [comment]
@npalm I have removed the ingress for now as it will also fail the linter (testing on another branch) due to missing dependencies.

File: templates/server-service.yaml (hunk 1 above)
Comment (npalm) [reply templates/server-service.yaml:11]: In that case I hope we get it working with the ALB, let's see.

**Pull request merged**